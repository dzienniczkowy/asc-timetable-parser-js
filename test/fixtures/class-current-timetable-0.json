{
  "r": {
    "ttitems": [
      {
        "type": "card",
        "date": "2022-10-31",
        "uniperiod": "1",
        "starttime": "08:00",
        "endtime": "09:45",
        "subjectid": "-22",
        "classids": [
          "-105"
        ],
        "groupnames": [
          ""
        ],
        "igroupid": "",
        "teacherids": [
          "-48"
        ],
        "classroomids": [
          "-28"
        ],
        "colors": [
          "#60FFC0"
        ],
        "durationperiods": 2
      },
      {
        "type": "card",
        "date": "2022-10-31",
        "uniperiod": "3",
        "starttime": "10:00",
        "endtime": "10:45",
        "subjectid": "-2",
        "classids": [
          "-105"
        ],
        "groupnames": [
          ""
        ],
        "igroupid": "",
        "teacherids": [
          "-161"
        ],
        "classroomids": [
          "-16"
        ],
        "colors": [],
        "removed": true
      },
      {
        "type": "card",
        "date": "2022-10-31",
        "uniperiod": "4",
        "starttime": "11:00",
        "endtime": "11:45",
        "subjectid": "-14",
        "classids": [
          "-105"
        ],
        "groupnames": [
          ""
        ],
        "igroupid": "",
        "teacherids": [
          "-37"
        ],
        "classroomids": [
          "-12"
        ],
        "colors": [
          "#66FFFF",
          "#220000"
        ]
      },
      {
        "type": "card",
        "date": "2022-10-31",
        "uniperiod": "5",
        "starttime": "12:00",
        "endtime": "13:45",
        "subjectid": "-43",
        "classids": [
          "-105"
        ],
        "groupnames": [
          "wychowanie fizyczne_ch"
        ],
        "igroupid": "",
        "teacherids": [
          "-74"
        ],
        "classroomids": [
          "-39"
        ],
        "colors": [
          "#003399"
        ],
        "durationperiods": 2,
        "cellSlices": "01",
        "cellOrder": 1
      },
      {
        "type": "card",
        "date": "2022-10-31",
        "uniperiod": "5",
        "starttime": "12:00",
        "endtime": "13:45",
        "subjectid": "-43",
        "classids": [
          "-107",
          "-105"
        ],
        "groupnames": [
          "wychowanie fizyczne_dz"
        ],
        "igroupid": "",
        "teacherids": [
          "-70"
        ],
        "classroomids": [
          "-64"
        ],
        "colors": [
          "#FF8080"
        ],
        "durationperiods": 2,
        "cellSlices": "10",
        "cellOrder": 0
      }
    ],
    "swapAxis": false,
    "rights": {
      "subjects": true,
      "classes": true,
      "teachers": true,
      "classrooms": true,
      "students": true,
      "igroups": true,
      "classroomsupervision": false,
      "teachers_summary": true,
      "classes_summary": true,
      "classrooms_summary": true,
      "igroups_summary": true
    },
    "_changeEvents": {
      "dbi:global_settings": 5037479
    }
  }
}
