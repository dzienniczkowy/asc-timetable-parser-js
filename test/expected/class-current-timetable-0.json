{
  "lessons": [
    {
      "type": "lesson",
      "date": "2022-10-31",
      "periodId": "1",
      "startTime": "08:00",
      "endTime": "09:45",
      "subjectId": "-22",
      "classIds": [
        "-105"
      ],
      "groupNames": [],
      "teacherIds": [
        "-48"
      ],
      "classroomIds": [
        "-28"
      ],
      "color": "#60FFC0",
      "removed": false
    },
    {
      "type": "lesson",
      "date": "2022-10-31",
      "periodId": "3",
      "startTime": "10:00",
      "endTime": "10:45",
      "subjectId": "-2",
      "classIds": [
        "-105"
      ],
      "groupNames": [],
      "teacherIds": [
        "-161"
      ],
      "classroomIds": [
        "-16"
      ],
      "color": null,
      "removed": true
    },
    {
      "type": "lesson",
      "date": "2022-10-31",
      "periodId": "4",
      "startTime": "11:00",
      "endTime": "11:45",
      "subjectId": "-14",
      "classIds": [
        "-105"
      ],
      "groupNames": [],
      "teacherIds": [
        "-37"
      ],
      "classroomIds": [
        "-12"
      ],
      "color": "#66FFFF",
      "removed": false
    },
    {
      "type": "lesson",
      "date": "2022-10-31",
      "periodId": "5",
      "startTime": "12:00",
      "endTime": "13:45",
      "subjectId": "-43",
      "classIds": [
        "-105"
      ],
      "groupNames": [
        "wychowanie fizyczne_ch"
      ],
      "teacherIds": [
        "-74"
      ],
      "classroomIds": [
        "-39"
      ],
      "color": "#003399",
      "removed": false
    },
    {
      "type": "lesson",
      "date": "2022-10-31",
      "periodId": "5",
      "startTime": "12:00",
      "endTime": "13:45",
      "subjectId": "-43",
      "classIds": [
        "-107",
        "-105"
      ],
      "groupNames": [
        "wychowanie fizyczne_dz"
      ],
      "teacherIds": [
        "-70"
      ],
      "classroomIds": [
        "-64"
      ],
      "color": "#FF8080",
      "removed": false
    }
  ]
}
